ci-operator/step-registry/cluster-profiles/cluster-profiles-config.yaml

```diff
@@ -456,6 +456,7 @@
         - gcp-filestore-csi-driver-operator
         - cluster-api-provider-gcp
         - cluster-etcd-operator
+        - kueue-operator
```

Comment:
In theory, install workflow does not rely on cluster profile, so it is supposed to use your own gcp cluster profile.

Reply:
I got an error saying `gcp-qe` is restricted only a few operators.

Reply:
Right, the restriction on `gcp-qe` cluster profile is on purpose. That is why I said your testing should use your own cluster profile, or some other unrestricted cluster profiles.